tests/AuthTestTraitTest.php

```diff
@@ -46,4 +48,19 @@ public function testActingWithEmptySession()
 
         $this->assertEmpty($loginSession);
     }
+
+    public function testActingAs()
+    {
+        $mockedUser = new MockAuthUser();
+        $mockedUser->someIntProperty = 0;
+
+        $this->actingAs($mockedUser);
+
+        $user = Auth::user();
+        $user->someIntProperty = 1;
+
+        $this->actingAs($mockedUser);
+
+        $this->assertEquals(0, Auth::user()->someIntProperty);
```

Comment:
let's try to test it like this

```suggestion
        $mockedUser = new MockAuthUser();

        $this->actingAs($mockedUser);

        $this-> assertNotSame($mockedUser, Auth::user());
```

Reply:
This work is great! Thanks.